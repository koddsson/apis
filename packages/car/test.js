/* global expect, it, describe, jest */

jest.mock('request');

const lookupCar = require('./index.js');

describe('car lookup', () => {
  it('should lookup cars by plate number', () => {
    return lookupCar('AA031').then(carData => {
      expect(carData.number).toEqual('AA031');
    });
  });
});