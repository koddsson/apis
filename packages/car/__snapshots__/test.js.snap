// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`car lookup should lookup cars by plate number 1`] = `
Object {
  "color": "Blár",
  "factoryNumber": "VF37ENFZE32286866",
  "nextCheck": "01.01.2017",
  "number": "AA031",
  "pollution": " g/km",
  "registeredAt": "26.02.1998",
  "registryNumber": "AA031",
  "status": "Í lagi",
  "subType": "306",
  "type": "PEUGEOT - 306 (Blár)",
  "weight": "1120 kg",
}
`;
